ts/parts/Tooltip.ts

```diff
@@ -257,6 +257,40 @@ var doc = H.doc,
     syncTimeout = H.syncTimeout,
     timeUnits = H.timeUnits;
 
+/**
```

Comment:
This file doesn't use private functions for anything else, I prefer keeping it consistent with the existing code, which means adding a class function. Then the arguments here are not needed, as they are part of the `Tooltip` context.

Reply:
I have made some relevant comments to this down below in one of your other comments.